src/binary/requests.ts

```diff
@@ -52,11 +52,13 @@ export function configuration(
   );
 }
 
-export function setState(state) {
+export function setState(state: any) {
```

Comment:
why any?

Reply:
Because I have no idea what types it accepts. You know?

Reply:
https://github.com/codota/acpl-paid/blob/791760a8a98a2823e717b71b16bb32d42983d69f/tabnine/server/src/api/v2_1.rs#L73

Reply:
Fucking shit 🤦🏻‍♂️